languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8820,6 +8836,25 @@ though, and has since been slightly modified)-->
           <message>Le nom exact est <suggestion>coronavirus</suggestion>. « Corona » est un abus de language.</message>
           <example correction="coronavirus">Depuis le <marker>corona</marker> ça n'est plus possible.</example>
       </rule>
+      <!-- this is an example rule: -->
+      <rule id="N_A_V_PPR" name="Participe présent au lieu d'adjectif">
```

Comment:
Shouldn't this have default=temp_off?

Reply:
I thought this had been tested already since I found it with default temp_off?